monitoring_test.go

```diff
@@ -86,8 +86,8 @@ func TestUnmarshalMonitoringJobs(t *testing.T) {
 		t.Error("Status is not up")
 	}
 	r := j.Rules[0]
-	assert.Equal(t, r.Key, "rtt", "RTT rule key is wrong")
-	assert.Equal(t, r.Value.(float64), float64(100), "RTT rule value is wrong")
+	// assert.Equal(t, r.Key, "rtt", "RTT rule key is wrong")
```

Comment:
nope that makes sense completely, if we can get the tests working with out it no need to add the dep :+1: